_includes/configuration/images-upload-credentials.md

```diff
@@ -1,14 +1,14 @@
-## images_upload_credentials
+## `images_upload_credentials`
 
-The **images_upload_credentials** option lets you specify if calls to the configured [`images_upload_url`](#images_upload_url) should pass along credentials (such as cookies, authorization headers, or TLS client certificates) for cross-domain uploads. When set to `true`, credentials will be sent to the upload handler, similar to the [`withCredentials` property of `XMLHttpRequest`s](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials).
+The **images_upload_credentials** option specifies if calls to the configured [`images_upload_url`](#images_upload_url) should pass along credentials (such as cookies, authorization headers, or TLS client certificates) for cross-domain uploads. When set to `true`, credentials will be sent to the upload handler, similar to the [`withCredentials` property of `XMLHttpRequest`s](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials).
```

Comment:
```suggestion
The **images_upload_credentials** option specifies whether calls to the configured [`images_upload_url`](#images_upload_url) should pass along credentials (such as cookies, authorization headers, or TLS client certificates) for cross-domain uploads. When set to `true`, credentials will be sent to the upload handler, similar to the [`withCredentials` property of `XMLHttpRequest`s](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials).
```

Thoughts?

Reply:
Accepted your suggestion (Thanks. I was too tired when I got to this one) https://github.com/tinymce/tinymce-docs/pull/1708/commits/bb8425e34457abc59a9ca725089672fc9f8c083b